### PR TITLE
Create homebrew formula to install latest release

### PR DESCRIPTION
**Pull request by liilac**
Directory structure is intended to allow use of this repository as a homebrew tap

See #105 #63

### CODE REVIEW

File: homebrew/Formula/loki-network.rb [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  homepage "https://loki.network"
+  url "https://github.com/loki-project/loki-network/archive/v0.3.1.tar.gz"
+  sha256 "a745cd156fd08be575f0e9f2fd56f4d6cde896363103c6f49499c90451021b34"
+  
+  depends_on "cmake" => :build
+  depends_on "rapidjson" => :recommended
+  depends_on "abyss" => :optional
+  depends_on "ninja" => :build
+  option "with-jsonrpc" "Build with JSON-RPC support"
+  option "without-avx2" "Build with AVX2 support"
```
Comment (majestrate) [inline homebrew/Formula/loki-network.rb]: this should be with-avx2 right?

File: homebrew/Formula/loki-network.rb [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    cmake_args = *std_cmake_args
+    cmake_args << "-GNinja"
+
+    system "cmake", "-B", "#{buildpath}/build", "-S", "#{buildpath}", *cmake_args
+    system "ninja", "-C", "#{buildpath}/build"
+    system "ninja", "-C", "#{buildpath}/build", "install"
+
+    bin.install "#{buildpath}/build/lokinet"
+    bin.install "#{buildpath}/lokinet-bootstrap"
+    bin.install "#{buildpath}/build/dns"
```
Comment (majestrate) [inline homebrew/Formula/loki-network.rb]: `dns` `llarpc` and possibly `rcutil` aren't needed

File: homebrew/Formula/loki-network.rb [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    bin.install "#{buildpath}/build/llarpc"
+    bin.install "#{buildpath}/build/rcutil"
+  end
+
+  test do
+    system "#{bin}/lokinet", "-h"
```
Comment (majestrate) [inline homebrew/Formula/loki-network.rb]: run the `#{buildpath}/build/testAll` binary instead

**Comment by liilac** [comment]
PR has been updated to take into account feedback.

**Comment by liilac** [comment]
See https://github.com/Homebrew/homebrew-core/pull/34939

**Comment by majestrate** [comment]
it's probably too early in the game to submit to mainline homebrew.

**Comment by majestrate** [comment]
On Mon, Dec 10, 2018 at 06:18:13AM -0800, Lilac wrote:
> @majestrate thoughts on whether you want me to withdraw my PR?  It was made yesterday.  Currently isn't being merged due to a test failure which I haven't fixed yet.
looks like they closed it already so it's your choise.

**Comment by majestrate** [comment]
any status updates on this?

**Comment by majestrate** [comment]
there doesn't seem to be any updates here for a while so i'll close this

**Comment by liilac** [comment]
Requesting this PR be re-opened and merged (directory structure may need to change now), as this is this is complete work, that was only not merged previously due to the intention for it to be maintained outside of the project, which didn't happen.

@neuroscr @majestrate

**Comment by majestrate** [comment]
On Mon, Apr 15, 2019 at 10:45:44PM -0700, Lilac wrote:
> Requesting this PR be re-opened and merged (directory structure may need to change now), as this is this is complete work, that was only not merged previously due to the intention for it to be maintained outside of the project, which didn't happen.
> 
> @neuroscr @majestrate 

this PR will need to be updated as it became stale.

**Comment by majestrate** [comment]
any updates @liilac ?

**Comment by majestrate** [comment]
this PR is way out of date.

**Pull request closed without merging**